libres/lib/analysis/ies/ies.cpp

```diff
@@ -68,7 +68,6 @@ auto logger = ert::get_logger("ies");
 #define ITER_KEY "ITER"
 #define IES_DEBUG_KEY "IES_DEBUG"
 
-#define IES_SUBSPACE_KEY "IES_SUBSPACE"
```

Comment:
What is it we are removing here?

Reply:
A `bool` flag `ies_subspace` which was never used. I am guessing it was first introduced as a `bool` - and then later promoted to the `enum inversion_type`